### PR TITLE
Add freesas recipe

### PR DESCRIPTION
**Pull request by beenje**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by beenje** [comment]
@conda-forge/help-python-c, ready for review!

I removed `pip check` because it failed on macOS with: `pyfai 2023.5.0 has requirement setuptools<60.0.0, but you have setuptools 68.1.2`
Looking at the code, I don't see why setuptools should be required at runtime. Neither `setuptools` nor `pkg_resources` is used (except in setup.py).

**Comment by beenje** [comment]
@kif if you are interested in being added as maintainer (now or later), just let me know.

**Comment by kif** [comment]
Hi Beenje,
Thanks for the packaging of `freesas`. I am not a guru in conda packaging but I am nevertheless interested in the topic.

For info, `freesas` will be migrated to meson-python when all packaging is OK (`fabio` is our guinea-pig).

**Comment by carterbox** [comment]
> I removed `pip check` because it failed on macOS with: `pyfai 2023.5.0 has requirement setuptools<60.0.0, but you have setuptools 68.1.2`
> Looking at the code, I don't see why setuptools should be required at runtime. Neither `setuptools` nor `pkg_resources` is used (except in setup.py).

Please open a PR in the pyfai recipe to fix this issue.

File: recipes/freesas/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/freesas-{{ version }}.tar.gz
+  sha256: 4b919ddea2d732a805a94b99482e68b3184f6348cfae52dec3c1fd0267dbcabd
+
+build:
```
Comment (carterbox) [inline recipes/freesas/meta.yaml:12]:
```suggestion
build:
  ignore_run_exports_from:
  # cxx standard library is unused
    - {{ compiler('cxx') }}
```

**Comment by beenje** [comment]
Thanks @carterbox !

**Pull request merged**